browser/debounce/debounce_browsertest.cc

```diff
@@ -164,72 +169,9 @@ IN_PROC_BROWSER_TEST_F(BraveSiteHacksNetworkDelegateBrowserTest,
   }
 }
 
-IN_PROC_BROWSER_TEST_F(BraveSiteHacksNetworkDelegateBrowserTest,
-                       QueryStringFilterSameSite) {
```

Comment:
The necessary functionality to make these tests pass appears to be in the debouncer as well:
https://github.com/brave/brave-core/pull/9065/files#diff-2078587030c3d53b4f82c2f9231091a39f759c18a544f68015b62a3aa0be4df9R162-R165

but it would be good to keep a test for this to prevent a regression of https://github.com/brave/brave-browser/issues/9020

Reply:
Reverted in https://github.com/brave/brave-core/commit/4aa1a0bf4d75ecc201119e062fdd1e55ec6e6ec4

The original regex-based code in Brave site hacks is untouched. We may revisit migration to the debounce infrastructure at a later time.